src/main/java/org/jalasoft/moi/controller/python/PythonController.java

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright (c) 2020 Jalasoft.
+ *
+ *  This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information"). You shall not disclose such Confidential
+ *  Information and shall use it only in accordance with the terms of the
+ *  license agreement you entered into with Jalasoft.
+ */
+
+package org.jalasoft.moi.controller.python;
+
+import org.jalasoft.moi.controller.CodeHelper;
+import org.jalasoft.moi.model.core.Params;
+import org.json.simple.parser.ParseException;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+
+/**
+ * This class defines the controller for Python.
+ *
+ * @version        1.0
+ * @author         Diego Perez
+ */
+@RestController("/py")
+public class PythonController {
```

Comment:
If you are put complete names for all languages this should follow the same convention:
use -> `/python`
instead -> `/py`